src/jquery.multiselect.js

```diff
@@ -579,71 +623,116 @@
       // restored to their defaultValue prop on form reset, and the reset
       // handler fires before the form is actually reset.  delaying it a bit
       // gives the form inputs time to clear.
-      $(this.element[0].form).on('reset.' + this._namespaceID, function() {
+      $(self.element[0].form).on('reset.' + self._namespaceID, function() {
         setTimeout($.proxy(self.refresh, self), 10);
       });
     },
 
     // Determines the minimum width for the button and menu
     // Can be a number, a digit string, or a percentage
     _getMinWidth: function() {
-      var minVal = this.options.minWidth;
+      var self = this;
+      var minWidth = self.options.minWidth;
       var width = 0;
-      switch (typeof minVal) {
+
+      switch (typeof minWidth) {
         case 'number':
-          width = minVal;
+          width = minWidth;
           break;
         case 'string':
-          var lastChar = minVal[ minVal.length -1 ];
-          width = minVal.match(/\d+/);
-          if(lastChar === '%') {
-            width = this.element.parent().outerWidth() * (width/100);      // element is a jQuery object
-          } else {
-            width = parseInt(minVal, 10);
-          }
-          break;
+         if (minWidth === 'auto') {
+            $menu = self.$menu;
+            var appendEl = self._getAppendEl();
+            var cssVisibility = $menu.css('visibility');
+            var cssWidth = $menu.css('width');
+            var cssDisplay = $menu.css('display');
+            var cssulDisplay = $menu.find('ul:eq(0)').css('display');
+            $menu.appendTo('body').css({visibility:'hidden', width:'auto', display:'inline'}).find('ul').css('display', 'inline');
+            var autoWidth = $menu.width();
+            $menu.css({visibility:cssVisibility, width:cssWidth, display:cssDisplay}).find('ul').css('display', cssulDisplay).appendTo( appendEl );
+            console.log('AUTO WIDTH: ', cssWidth, autoWidth);
```

Comment:
Delete the console statement.

Reply:
Oops!  Sorry about that.  Will remove.  

Also, I goofed as I did not intend to include this auto width functionality in this.   I am still fiddling with it.  I will remove it, also.  (sorry)